sys/include/net/gnrc/ipv6/nib.h

```diff
@@ -178,8 +184,109 @@ extern "C" {
  * context is a valid default router entry representing the router.
  */
 #define GNRC_IPV6_NIB_RTR_TIMEOUT           (0x4fcdU)
+
+/**
+ * @brief   Recalculate reachability timeout time.
+ *
+ * This message type is for the event of recalculating the reachability timeout
+ * time. The expected message context is a valid [interface](_nib_iface_t).
```

Comment:
no need for `@ref` here?

Reply:
Don't know why this reference is there anyway... #7456 moves that to `gnrc_netif2_t` and `_nib_iface_t` is internal and thus not in the generated doc...